corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -665,6 +665,18 @@ Formplayer.Utils.touchformsError = function (message) {
     return Formplayer.Errors.GENERIC_ERROR + message;
 };
 
+Formplayer.Utils.reloginErrorHtml = function () {
+    var isWebApps = FormplayerFrontend.request('currentUser').environment === FormplayerFrontend.Constants.WEB_APPS_ENVIRONMENT;
+    if (isWebApps) {
+        var url = hqImport("hqwebapp/js/initial_page_data").reverse('login_new_window');
+        return _.template(gettext("Looks like you got logged out because of inactivity, but your work is safe. " +
+                                  "<a href='<%= url %>' target='_blank'>Click here to log back in.</a>"))({url: url});
```

Comment:
Do you think adding something like "and then return to this window once you have" would make people less likely to try to reload this page (thus losing their work unless they also know about Incomplete Forms)?

Reply:
Possibly. There's definitely a balance to strike here between simplicity and clarity. @djmore9 can you suggest working?